cancunDevs/metabot
# Feat - Add .env.sample file, sample request

#### What does this PR do?
This PR includes a sample request for bot, and the `.env.sample` file to store our environment variables.

#### Where should the reviewer start?
* `index.js`
* `src/bot.js`
* `.env.sample`

#### How should this be manually tested?
Clone this repo, create `.env` file, fill the variables with the sample data, and run, it should deploy in your localhost in the selected port.
